lib/std/os.zig

```diff
@@ -3199,8 +3199,16 @@ pub fn isatty(handle: fd_t) bool {
         if (isCygwinPty(handle))
             return true;
 
-        var out: windows.DWORD = undefined;
-        return windows.kernel32.GetConsoleMode(handle, &out) != 0;
+        // Support for ANSI escapes sequences was added to Windows 10 1511
+        // on 2016.
```

Comment:
The idea was to *advertising* this feature, but probably it is not necessary.

A version check is not necessary, since if `SetConsoleMode` returns `0`, then the feature is not supported.